schema/google/showcase/v1beta1/compliance_suite.json

```diff
@@ -0,0 +1,40 @@
+{
+  "group": [
+    {
+      "name": "Fully working conversions",
+      "rpcs": ["Compliance.RepeatDataBody", "Compliance.RepeatDataQuery", "Compliance.RepeatDataSimplePath"],
+      "requests": [
+	{
+	  "name": "Basic data types",
+	  "info": {
+            "f_string": "Hello"
+	  }
+	},
+	{
+	  "name": "Strings with spaces",
+	  "info": {
+            "f_string": "Hello there"
+	  }
+	},
+	{
+	  "name": "Strings with quotes",
```

Comment:
Do we care about weird edge cases like emojis?

Reply:
We will. These are just very basic test cases for well-formed inputs. I plan to expand them in subsequent PRs.